examples/cluster/cluster.yaml

```diff
@@ -2,11 +2,17 @@
 apiVersion: cluster.x-k8s.io/v1alpha2
 kind: Cluster
 metadata:
+  annotations:
+    cluster.k8s.io/network-cni: ${NETWORK}
```

Comment:
```suggestion
    cluster.x-k8s.io/network-cni: ${NETWORK}
```

Reply:
We need to adopt the alpha `x-k8s.io` annotations for now, but otherwise good to go.

Reply:
> We need to adopt the alpha `x-k8s.io` annotations for now, but otherwise good to go.

thx very much.